code/modules/paperwork/fax.dm

```diff
@@ -6,6 +6,7 @@ GLOBAL_VAR_INIT(nt_fax_department, pick("NT HR Department", "NT Legal Department
 	icon = 'icons/obj/machines/fax.dmi'
 	icon_state = "fax"
 	density = TRUE
+	anchored_tabletop_offset = 6
```

Comment:
This is a var?? What the fuck?
I could've done this with the library computers, oh well.

Reply:
I wanted to set this to 6 for all objs by default, but there are directional things that break when they're placed on walls next to tables, so you need to remember about it when when you add a machine, yeah...